packages/core/admin/admin/src/components/MainNav/NavUser.tsx

```diff
@@ -70,25 +106,41 @@ export const NavUser = ({ children, initials, ...props }: NavUserProps) => {
       <Menu.Root>
         <MenuTrigger endIcon={null} fullWidth justifyContent="center">
           <Avatar.Item delayMs={0} fallback={initials} />
-          <VisuallyHidden tag="span">{children}</VisuallyHidden>
+          <VisuallyHidden tag="span">{displayName}</VisuallyHidden>
         </MenuTrigger>
-        <MenuContent popoverPlacement="top-center" zIndex={3}>
+
+        <MenuContent popoverPlacement="top-start" zIndex={3}>
+          <UserInfo direction="column" gap={0} alignItems="flex-start">
+            <Typography variant="omega" fontWeight="bold" textTransform="none">
+              {displayName}
+            </Typography>
+            <StyledTypography variant="pi" textColor="neutral600">
+              {user?.email}
+            </StyledTypography>
+            <BadgeWrapper>
+              {user?.roles?.map((role) => <Badge key={role.id}>{role.name}</Badge>)}
+            </BadgeWrapper>
+          </UserInfo>
+
+          <StyledMenuSeparator />
```

Comment:
```suggestion
          <Menu.Separator />
```

I don't think you need to create a custom component, you can directly use  `<Menu.Separator />`.

It means you can delete this part as well:

```
const StyledMenuSeparator = styled(Menu.Separator)`
  margin: 0;
  height: 1px;
  background: ${({ theme }) => theme.colors.neutral150};
`;
```